client/src/pages/Interview.js

```diff
@@ -52,6 +63,9 @@ const Interview = () => {
   const fetchInterview = useCallback(async () => {
     try {
       const { data } = await getInterview(roomId);
+      if (data.interview.code !== '') {
+        setValue(data.interview.code);
```

Comment:
should this be setOldCode? If not, where is oldCode being set?

Reply:
I thought I got rid of oldcode useState.  It's not being set anywhere